src/utils/events/formatEventFromServer.js

```diff
@@ -0,0 +1,69 @@
+import { formatEventTypeFromServer, formatOwnerFromServer } from './index';
+
+/**
+ * Normalizes the event object returned from the API server
+ *
+ * @param {Object} input
+ * @param {booelan} input.allow_others_to_trigger Whether or not non-owners can trigger the event
+ * @param {string} input.created_at ISO 8601 Extended Format date/time string
+ * @param {string} input.deleted_at ISO 8601 Extended Format date/time string
+ * @param {Object} input.EventType
+ * @param {string} input.EventType.client_id The ID of the client to which the event type belongs
+ * @param {string} input.EventType.created_at ISO 8601 Extended Format date/time string
+ * @param {string} input.EventType.description A description of the event type
+ * @param {string} input.EventType.id UUID corresponding with the event type
+ * @param {string} input.EventType.is_realtime_enabled
+ * @param {number} input.EventType.level The level of the event type
+ * @param {string} input.EventType.name The name of the event type
+ * @param {string} input.EventType.slug The slug of the event type
+ * @param {string} input.EventType.updated_at ISO 8601 Extended Format date/time string
+ * @param {string} input.event_type_id UUID corresponding with the event type
```

Comment:
For my own curiosity, will this be the same as `input.EventType.id`?

Reply:
Yeah, it's the same, but that's how it comes back from the API.